CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.2.3
+
+### Bug fixes
+
+* Fixed memory leak in `Java_io_realm_internal_Table_nativeSetPrimaryKey` (#3993).
```

Comment:
More like "when creating primary key metadata"?. This leak only occurs when setting up the primary key table.